inventory/test-allinone-ha.yaml

```diff
@@ -44,47 +44,45 @@ all:
             zuul-scheduler-slave:
               hosts: allinone02
               vars:
-                service_state: stopped
+                zuul_service_zuul_scheduler_state: stopped
         zuul-executor:
           children:
             zuul-executor-master:
               hosts: allinone01
             zuul-executor-slave:
               hosts: allinone02
               vars:
-                service_state: stopped
+                zuul_service_zuul_executor_state: stopped
         zuul-web:
           children:
             zuul-web-master:
               hosts: allinone01
             zuul-web-slave:
               hosts: allinone02
               vars:
-                service_state: stopped
+                zuul_service_zuul_web_state: stopped
         zuul-merger:
           children:
             zuul-merger-master:
               hosts: allinone01
             zuul-merger-slave:
               hosts: allinone02
               vars:
-                service_state: stopped
+                zuul_service_zuul_merger_state: stopped
         zuul-fingergw:
           children:
             zuul-fingergw-master:
               hosts: allinone01
             zuul-fingergw-slave:
               hosts: allinone02
               vars:
-                service_state: stopped
+                zuul_service_zuul_fingergw_state: stopped
         zuul-log-server:
           children:
             logserver-master:
               hosts: allinone01
             logserver-slave:
               hosts: allinone02
-              vars:
-                service_state: stopped
```

Comment:
Do we have to change the var to logserver_state instead delete the var directly?

Reply:
this is used by unison, not we do not need this when using rsync to sync logs.